tests/ui-internal/custom_ice_message.rs

```diff
@@ -10,5 +10,6 @@
 #![allow(clippy::missing_clippy_version_attribute)]
 
 fn it_looks_like_you_are_trying_to_kill_clippy() {}
+//~^ ice: Would you like some help with that?
```

Comment:
I love that the error message is now also there. This makes the joke complete :tada: